releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.AspNetCore.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
```

Comment:
We don't have a release a 4.0.0 to deprecate it, right? So just have the following at the beginning of this release note.

## Deprecation

This package has been deprecated and it's functionality is now available as a part of the [Microsoft.FeatureManagement.ApplicationInsights](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.md) package.

Reply:
Agreed